src/defineNitroTRPCEventHandler.ts

```diff
@@ -83,7 +88,7 @@ export type NitroRequestHandler = <TRouter extends AnyRouter>({
   createContext?: CreateContextFn<TRouter>
   responseMeta?: ResponseMetaFn<TRouter>
   onError?: OnErrorFn<TRouter>
-}) => EventHandler<string | undefined>
+}) => EventHandler<TRequest, EventHandlerResponse<string | undefined>>
```

Comment:
@danielroe Sufficient that the generic response type that would usually be passed as `TResponse` is just string or undefined, to follow tRPC guidelines on the "stringified" response.